uncorrelate/statistical_inefficiency_dhdl.py

```diff
@@ -51,10 +51,7 @@ def uncorrelate(self, dfs, lower):
 
         uncorrelated_dfs = []
         for dhdl_, l, df in zip(self.dhdls, dl, dfs):
-            ind = np.array(l, dtype=bool)
-            ind = np.array(ind, dtype=int)
-            dhdl_sum = dhdl_.dot(ind)
```

Comment:
What's the rationale for not using `dhdl_sum`?

Reply:
I can't recall why this change was made.... I will have to look into this.

Reply:
This was a temporary change due to https://github.com/alchemistry/alchemlyb/issues/166. We identified the cause just now. I reverted the change.